components/Home/Bonding/requests.jsx

```diff
@@ -178,20 +183,16 @@ const getLpTokenNamesForProducts = async (productList, events) => {
 };
 
 const getCurrentPriceBalancer = async (tokenAddress) => {
-  const { lpChainId, poolId } = await getLpTokenDetails(
-    tokenAddress,
-  );
+  const { lpChainId, poolId } = await getLpTokenDetails(tokenAddress);
 
   const balancerConfig = { network: lpChainId, rpcUrl: RPC_URLS[lpChainId] };
   const balancer = new BalancerSDK(balancerConfig);
 
   const pool = await balancer.pools.find(poolId);
   const totalSupply = pool.totalShares;
-  const reservesOLAS = pool.tokens[0].balance * 1.0;
-  // TODO: ADDRESSES[lpChainId].olasAddress is capital, .address is small
-  // (pool.tokens[0].address !== ADDRESSES[lpChainId].olasAddress
-  // ? pool.tokens[1].balance
-  // : pool.tokens[0].balance) * 1.0;
+  const reservesOLAS = (areAddressesEqual(pool.tokens[0].address, ADDRESSES[lpChainId].olasAddress)
+    ? pool.tokens[0].balance
+    : pool.tokens[1].balance) * 1.0;
 
   // TODO: where do we multiply the price by 2? somewhere after?
```

Comment:
Please remove this line as you have moved things around and now it's in the same file